setup.py

```diff
@@ -1,82 +1,64 @@
 import os
-import re
 import sys
 import platform
 import subprocess
 
-from setuptools import setup, find_packages, Extension
+from setuptools import setup, Extension
 from setuptools.command.build_ext import build_ext
-from distutils.version import LooseVersion
 
 
 class CMakeExtension(Extension):
     def __init__(self, name):
         Extension.__init__(self, name, sources=[])
 
+
 class CMakeBuild(build_ext):
     def build_extension(self, ext):
-
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
 
         debug = int(os.environ.get("DEBUG", 0)) if self.debug is None else self.debug
-        cfg = 'Debug' if self.debug else 'Release'
-        print('building CMake extension in %s configuration' % cfg)
+        cfg = "Debug" if self.debug else "Release"
+        print("building CMake extension in %s configuration" % cfg)
 
         cmake_args = [
```

Comment:
Maybe I am doing something wrong, but with `pip install .` it does not work because `pybind` is disabled, and I need to `pip install . --config-settings="-DFETCH_PYBIND=ON"`... Should we have it active by default?